frontend/src/utilities/useWatchDashboardConfig.tsx

```diff
@@ -3,6 +3,7 @@ import { DashboardConfig } from '../types';
 import { POLL_INTERVAL } from './const';
 import { useDeepCompareMemoize } from './useDeepCompareMemoize';
 import { fetchDashboardConfig } from '../services/dashboardConfigService';
+import { logout } from './appUtils';
```

Comment:
```suggestion
import { logout } from '../app/appUtils';
```

Diff upstream: https://github.com/opendatahub-io/odh-dashboard/pull/532/commits/a72cf5272191775b66d3bd2e8e9eb2eb4dd21bd1